test/index.ts

```diff
@@ -282,3 +282,22 @@ describe('user query', () => {
     expect(query.data.errors).to.be.deep.eq([userDoesntExistError]);
   });
 });
+
+describe('user list query', () => {
+  it('length lower or equal arg quantity', async () => {
```

Comment:
uma sugestão para nomear os its é utilizar o padrao `should return something for case a, b, c`  confesso que lendo as descrições tenho dificuldade para entender o que vc quer testar